README.md

```diff
@@ -283,6 +283,16 @@ Here are some examples of conditions you can use:
 | `[DOMAIN_EXPIRATION]`      | Resolves into the duration before the domain expires (valid units are "s", "m", "h".)     | `24h`, `48h`, `1234h56m78s`                  |
 | `[DNS_RCODE]`              | Resolves into the DNS status of the response                                              | `NOERROR`                                    |
 
+#### Severity
+
+For failure case you can specify index of severity for each condition. It could be an integer from 1 to 10. Example of usage: `10::[STATUS] == 200`. For case, when severity index is omitted, severity with index `10` will be used.
+
+| Total severity index count | Severity status |
+|:---------------------------|:----------------|
+|`0`                         |`0` (None)       |
+|`1-4`                       |`1` (Low)        |
+|`5-8`                       |`2` (Medium)     |
+|`9..`                       |`3` (High)       |                                 |
```

Comment:
I'm really not a fan of this syntax. I'm not saying I have a better suggestion, but I'd like to let this sit for a bit while I think about it.

Also, the numbers seem very random to me. If they get translated to None/Low/Medium/High anyways, why don't we just replace the number by the value? And why is the default `10`?

I think something like
```yaml
conditions:
  - "Low :: [RESPONSE_TIME] < 100"
  - "High :: [CONNECTED] == true"
```
would make more sense.

Reply:
Hi, @TwiN! Thanks for your response! I'll update this PR on next week following this comments and suggestions.

Reply:
@TwiN I have updated my PR with your recommendations. Please, take a look!